code/modules/jobs/job_types/ostia.dm

```diff
@@ -0,0 +1,273 @@
+
+/datum/job/CaesarsLegiontown/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/bloodleaf)
+	
+
+/datum/job/CaesarsLegiontown/subject
+	title = "Legion Subject"
+	flag = F13OSTIASUBJECT
+	department_flag = OSTIA
+	total_positions = -1
+	spawn_positions = -1
+	description = "A citizen of the Legion port of Ostia. You are not a slave, but freedom is a concept quite far from you. You are not part of the legion military (ain't a off duty role), and live a comfy life. You can manage the shop, the café, or just help the legion arround. You can also become a Gladiator in the arena. As a subject, you must have a latin name."
+	supervisors = "You obey the governor, and the legion military, whom you are also loyal too."
+	display_order = JOB_DISPLAY_ORDER_F13OSTIASUBJECT 
+	exp_requirements = 0
+	outfit = /datum/outfit/job/CaesarsLegiontown/subject
+
+	loadout_options = list(
+		/datum/outfit/loadout/subjectoflegion,
+		/datum/outfit/loadout/legionshopkeeper,
+		/datum/outfit/loadout/legiongladiator,
+		/datum/outfit/loadout/legionbarkeep,
+		)
+
+	matchmaking_allowed = list(
+		/datum/matchmaking_pref/friend = list(
+			/datum/job/CaesarsLegiontown/slave,
+		),
+		/datum/matchmaking_pref/rival = list(
+			/datum/job/CaesarsLegiontown/slave,
+		),
+	)
+
+	access = list(ACCESS_BAR, ACCESS_TOWN)
+	minimal_access = list(ACCESS_BAR, ACCESS_TOWN)
+
+/datum/outfit/job/CaesarsLegiontown/subject/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+	if(visualsOnly)
+		return
+	ADD_TRAIT(H, TRAIT_MARS_TEACH, src)
+
+/datum/outfit/job/CaesarsLegiontown/subject
+	name = "Legion Subject"
+	jobtype = /datum/outfit/job/CaesarsLegiontown/subject
+	id = /obj/item/card/id/dogtag/legimmune
+	uniform = /obj/item/clothing/under/civ/roman_centurion
+	shoes =	/obj/item/clothing/shoes/roman
+	ears = /obj/item/radio
+
+
+/datum/outfit/loadout/subjectoflegion
+	name = "Legion Subject"
+	backpack_contents = list(		
+		/obj/item/ammo_box/a357 = 1,
+		/obj/item/gun/ballistic/revolver/colt357 = 1,
+		/obj/item/stack/f13Cash/random/denarius/low = 2,
+		/obj/item/flashlight/lantern = 1,
+	)
+
+/datum/outfit/loadout/legiongladiator
+	name = "Legion Gladiator"
+	backpack_contents = list(
+		/obj/item/clothing/head/helmet/gladiator = 1,
+		/obj/item/clothing/under/gladiator = 1,
+		/obj/item/clothing/shoes/roman = 1,
+		/obj/item/flashlight/lantern = 1,
+		/obj/item/melee/onehanded/machete = 1,
+		/obj/item/stack/f13Cash/random/denarius/low = 1,
+		/obj/item/book/granter/trait/bigleagues = 1,
+		/obj/item/shield/riot/buckler = 1,
+		/obj/item/lighter = 1,
+		)
+
+/datum/outfit/loadout/legionbarkeep
+	name = "Legion Restaurant"
```

Comment:
I would suggest a name like "Legion Innkeep" or similar, since a Restaurant in English is a building you eat in.

Reply:
It was supposed to be just that, a restaurant. But a Inn might be even better. Ill modify the building to be a inn.

Reply:
Oh I figured it was, though "Restaurant" is not something you'd call someone is all.
I can make the corrections I've noted if it'd be easier for you.